src/utils/addInitContainerCommand.ts

```diff
@@ -0,0 +1,36 @@
+import { addInitContainer } from "@socialgouv/kosko-charts/utils/addInitContainer";
+import { ok } from "assert";
+import type { Deployment } from "kubernetes-models/apps/v1/Deployment";
+import { Container } from "kubernetes-models/v1/Container";
+
+export const addInitContainerCommand = (
+  deployment: Deployment,
+  container: Partial<Container>
+) => {
+  ok(deployment.spec);
+  ok(deployment.spec.template);
+  ok(deployment.spec.template.spec);
+  const image = deployment.spec.template.spec.containers[0].image;
+  const initContainer = new Container({
+    args: [],
+    command: ["hostname"],
+    image,
+    imagePullPolicy: "Always",
+    name: "init",
```

Comment:
we might want to use the command first element as name by default for clarification

Reply:
`container.command[0]` ?

Reply:
yes